internal/build/test_utils.go

```diff
@@ -174,6 +175,27 @@ func (f *dockerBuildFixture) assertImageNotExists(ref reference.NamedTagged) {
 	}
 }
 
+func (f *dockerBuildFixture) assertImageHasLabels(ref reference.Named, expected map[string]string) {
+	inspect, _, err := f.dCli.ImageInspectWithRaw(f.ctx, ref.String())
+	if err != nil {
+		f.t.Fatalf("error inspecting image %s: %v", ref.String(), err)
+	}
+
+	if inspect.Config == nil {
+		f.t.Fatalf("'inspect' result for image %s has nil config", ref.String())
+	}
+
+	actual := inspect.Config.Labels
+	for k, expectV := range expected {
```

Comment:
Since we are only adding one label right now this could be rewritten more simply as (pseudocode):

```
if len(actual) == 1 && require.Contains(builtByTilt) {
```

Reply:
You're totally right but since this is here i guess i might as well leave it for future? 🤷‍♀